public/app/plugins/datasource/loki/datasource.test.ts

```diff
@@ -629,13 +633,3 @@ function makeAnnotationQueryRequest(options: any): AnnotationQueryRequest<LokiQu
     rangeRaw: timeRange,
   };
 }
-
```

Comment:
I have removed this as we removed running of different queries for Loki v0 and Loki  v1. It was removed while ago and therefore these tests are not necessary anymore.